File: app/src/main/java/com/chooloo/www/callmanager/ui/page/PageMvpPresenter.java
package com.chooloo.www.callmanager.ui.page;

import com.chooloo.www.callmanager.ui.base.MvpPresenter;

public interface PageMvpPresenter<V extends PageMvpView> extends MvpPresenter<V> {
    void onDialNumberChanged(String number);

    void onSearchTextChanged(String text);

    void onScrollStateChanged(int state);
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/page/PageMvpPresenter.kt
package com.chooloo.www.callmanager.ui.page

import com.chooloo.www.callmanager.ui.base.MvpPresenter

interface PageMvpPresenter<V : PageMvpView> : MvpPresenter<V> {

    fun onDialNumberChanged(number: String)
    fun onSearchTextChanged(text: String)
    fun onScrollStateChanged(state: Int)
}